app/js/app.js

```diff
@@ -35,6 +35,38 @@ window.addEventListener('DOMContentLoaded', () => {
     console.error('[Customizer] An error occurred getting the manifestURL');
   };
 
+  /**
+   * XXX: Bug 1148218 - [Customizer] Undo inline stylesheets from Bug 1148139
```

Comment:
This should be a ```TODO```. ```XXX``` comments are used for drawing attention to important information, usually without a bug filed.